README.md

```diff
@@ -1,14 +1,39 @@
+[![Published on NPM](https://img.shields.io/npm/v/@polymer/iron-form-element-behavior.svg)](https://www.npmjs.com/package/@polymer/iron-form-element-behavior)
 [![Build status](https://travis-ci.org/PolymerElements/iron-form-element-behavior.svg?branch=master)](https://travis-ci.org/PolymerElements/iron-form-element-behavior)
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://webcomponents.org/element/@polymer/iron-form-element-behavior)
 
-_[Demo and API docs](https://elements.polymer-project.org/elements/iron-form-element-behavior)_
+## IronFormElementBehavior
+`IronFormElementBehavior` adds a `name`, `value` and `required` properties to
+a custom element. It mostly exists for back compatibility with Polymer 1.x, and
+is probably not something you want to use.
```

Comment:
I think you should phrase the backwards compability part more aggressively so that people don't consider adding this to something that they didn't port all the way from Polymer 1. (e.g. `mostly` -> `only`) Maybe even add a 'no contributions will be accepted' note if you feel like it. :)

Reply:
✅